### PR TITLE
SSCS-10286 Add new state and event.

### PR DESCRIPTION
**Pull request by dattipoeHMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10286

Common change:
https://github.com/hmcts/sscs-common/pull/1226

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json [hunk 1]
```diff
@@ -14,5 +12,7 @@
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequest", "UserRole": "caseworker-sscs", "CRUD": "R"},
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs-superuser", "CRUD": "R"},
-  {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs", "CRUD": "R"}
+  {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs", "CRUD": "R"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseEventID": "listingError", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
+  {"LiveFrom": "10/06/2020", "CaseTypeID": "Benefit", "CaseEventID": "listingError", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"}
```
Comment (osheaf01) [inline benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json:17]: Very minor and optional: Change both dates on the new additions to today's date?

File: benefit/data/sheets/AuthorisationCaseState/AuthorisationCaseState-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[
+    {"LiveFrom": "27/06/2019", "CaseTypeID": "Benefit", "CaseStateID": "handlingError", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
+    {"LiveFrom": "27/06/2019", "CaseTypeID": "Benefit", "CaseStateID": "handlingError", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
+    {"LiveFrom": "27/06/2019", "CaseTypeID": "Benefit", "CaseStateID": "handlingError", "UserRole": "citizen", "CRUD": "R"},
+    {"LiveFrom": "25/06/2019", "CaseTypeID": "Benefit", "CaseStateID": "handlingError", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
```
Comment (osheaf01) [inline benefit/data/sheets/AuthorisationCaseState/AuthorisationCaseState-nonprod.json:5]: Very minor: today's date?

**Review by osheaf01 — APPROVED** [review]
LGTM

**Review by nickwest1 — APPROVED** [review]

File: benefit/data/sheets/CaseEvent/CaseEvent.json [hunk 3]
```diff
@@ -2516,13 +2516,14 @@
     "SecurityClassification": "Public"
   },
   {
-    "LiveFrom": "01/01/2018",
+    "LiveFrom": "10/06/2022",
     "CaseTypeID": "Benefit",
-    "ID": "createTestCase",
-    "Name": "Create test case",
-    "Description": "Create a test case in validAppealCreated state",
```
Comment (ss-evoco) [inline benefit/data/sheets/CaseEvent/CaseEvent.json:2523]: Do we not need this anymore ?

**Review by ss-evoco — APPROVED** [review]

**Review by MohammedHMCTS — APPROVED** [review]
Looks good to me remember to do a Version bump

File: benefit/data/sheets/CaseEvent/CaseEvent.json (hunk 3 above)
Comment (dattipoeHMCTS) [reply benefit/data/sheets/CaseEvent/CaseEvent.json:2523]: I think caseevent is too large now, the last events have their dates parsed incorrectly. It's a test event added years ago so I assume it's safe to remove

**Pull request merged**